cmd/fyne_demo/FyneApp.toml

```diff
@@ -2,4 +2,4 @@
   Icon = "../../theme/icons/fyne.png"
   Name = "Fyne Demo"
   ID = "io.fyne.demo"
-  Build = 6
```

Comment:
As this is not releasing a new version of these apps I don't think the build should be bumped...

Reply:
Oh indeed.